src/test/java/com/edmunds/tools/databricks/maven/ImportWorkspaceMojoTest.java

```diff
@@ -17,36 +17,45 @@
 package com.edmunds.tools.databricks.maven;
 
 import com.edmunds.rest.databricks.request.ImportWorkspaceRequest;
+import java.io.File;
 import org.apache.maven.plugin.MojoExecutionException;
+import org.mockito.Matchers;
 import org.mockito.Mockito;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import java.io.File;
 
-import static org.testng.Assert.fail;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.times;
 
-public class ImportWorkspaceMojoTest extends BaseDatabricksMojoTest {
+public class ImportWorkspaceMojoTest extends DatabricksMavenPluginTestHarness {
 
-    private ImportWorkspaceMojo underTest = new ImportWorkspaceMojo();
 
-    @BeforeMethod
-    public void init() throws Exception {
-        super.init();
+    private final String GOAL = "import-workspace";
 
-        underTest.setDatabricksServiceFactory(databricksServiceFactory);
-        underTest.setEnvironment("QA");
+    @BeforeClass
+    public void initClass() throws Exception {
+        super.setUp();
+    }
+
+    @BeforeMethod
+    public void beforeMethod() throws Exception {
+        super.beforeMethod();
     }
 
     @Test
     public void testImportWorksWithProperPath() throws Exception {
+        ImportWorkspaceMojo underTest = getNoOverridesMojo(GOAL);
+
         File workspacePath = new File(this.getClass().getResource("/notebooks")
-            .getPath());
+                .getPath());
+
         underTest.validate = true;
         underTest.setSourceWorkspacePath(workspacePath);
-        Mockito.when(project.getGroupId()).thenReturn("com.edmunds.test");
-        Mockito.when(project.getArtifactId()).thenReturn("mycoolartifact");
-        underTest.setProject(project);
+
+        Mockito.verify(libraryService, times(0)).install(Matchers.anyString(), Matchers.any());
```

Comment:
your call, do you want to match against the exact string here?

Reply:
This was just the existing test. I could try to update it.

Reply:
Looking at this a little closer, I'm not even sure why the import workspace mojo test is verifying that the install mojo was not called. Why? 

Unless you can think of a good reason, I'm just going to delete this verification.

Reply:
lol uh oh. Why is it checking against the library service :( please remove